src/search_renderer.cpp

```diff
@@ -136,10 +134,9 @@ std::string SearchRenderer::getHtml()
   allData.set("searchPattern", kiwix::encodeDiples(this->searchPattern));
   allData.set("searchPatternEncoded", urlEncode(this->searchPattern));
   allData.set("resultStart", to_string(resultStart + 1));
-  allData.set("resultEnd", to_string(min(resultEnd, estimatedResultCount)));
-  allData.set("resultRange", to_string(resultCountPerPage));
+  allData.set("resultEnd", to_string(min(resultStart + pageLength, estimatedResultCount)));
```

Comment:
`resultStart` and `resultEnd` are only used in the template to know how many result is displayed (on the total number of result).
It may be better to pass this number using a specific variable name.
("pageLength" could be a good name, but it must not be mixed with the requested page length.
The current page length can be different than the request page length if there is less result than requested or if we are at the last page and it contains less result that the requested length).

Reply:
@mgautierfr so there should be a `resultEnd` variable for this instead of `resultStart + pageLength`

Reply:
No, it was not my comment.
My comment was that the two tokens `resultEnd` and `resultStart` was only used in the template to calculate the different between two to get the number of result displayed in the page. And so we could calculate this is cpp and pass it directly to the token.

But I was wrong. The token doesn't calculate the different but display the both values `{{resultStart}}-{{resultEnd}}`. I was a bit quick on this point.

So there was nothing to change here. However, the change you've made to reintroduce `resultEnd` variable is ok, even if I was not asking for that. So we are good :)